src/ResourceManager/Automation/Commands.Automation/Commands.Automation.Netcore.csproj

```diff
@@ -29,6 +29,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Automation" Version="3.4.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="2.0.0-preview" />
```

Comment:
Keep in mind you won't be able to use this when merging back to our main branch.